plugins/scaffolder-backend/api-report.md

```diff
@@ -610,6 +615,8 @@ export interface TaskBroker {
   // (undocumented)
   claim(): Promise<TaskContext>;
   // (undocumented)
+  closeStaleTasks(): Promise<void>;
```

Comment:
Had to make this little tweak in the `createRouter`: 
```javascript
let databaseTaskStore: DatabaseTaskStore;
if (!options.databaseTaskStore) {
  databaseTaskStore = await DatabaseTaskStore.create({ database });
} else {
  databaseTaskStore = options.databaseTaskStore;
}
```

so that later I could execute the scheduler like: `if (databaseTaskStore.shutdownTask) { ... }`

The options I had to move the logic to the `router` 🤔 :
* Instantiate once more `databaseTaskStore` in here to pass it to the `backend-task`
* Change `taskBroker` so that I could extract the `databaseTaskStore` from it
* Passing `databaseTaskStore` also within the `createRouter` options - Which was what I did on the last commit